<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleLightControlWidget</class>
 <widget class="QWidget" name="SingleLightControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>99</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="lightGroupBox">
     <property name="title">
      <string>Light</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="onCheckBox">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSlider" name="lightLevel">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lightLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lightLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
